docs/modular-resources/upload/_index.md

```diff
@@ -145,31 +152,11 @@ Edit and then upload the `meta.json` file to set important configuration informa
 
    On a successful update, the command will return a link to the updated module in the Viam registry.
 
-5. Package your custom module to get it ready to upload to the Viam registry.
-   Currently, the Registry only supports `tar.gz` or `tar.xz` format.
-   Use the command below specific for the language of your module:
-
-   - To package a module written in Go, run the following commands from the same directory as your `meta.json` file:
-
-     ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-     go build -o bin/module ./module/main.go
-     tar -czf module.tar.gz bin/module
-     ```
-
-     For more information, see [Compile a module into an executable](/modular-resources/create/#compile-the-module-into-an-executable).
-
-   - To package a module written in Python, run the following command from the same directory as your `meta.json` file:
-
-     ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-     tar -czf module.tar.gz run.sh requirements.txt src
-     ```
```

Comment:
We also dont want python users to necessarily zip up their whole module directory because they probably have things like their python virtual environments in the directory (which will cause issues on other computers)

Reply:
Great point, this makes perfect sense. Retaining portions of this, in Python context only. Thank you for the helpful comment!!